server/store/datastore/migration/026_convert_to_new_pipeline_errors_format.go

```diff
@@ -15,11 +15,16 @@
 package migration
```

Comment:
Waiting for user feedback ...

Reply:
I can try this PR on my k8s install and report back?